src/testTrackConfig.js

```diff
@@ -10,6 +11,10 @@ var TestTrackConfig = (function() { // jshint ignore:line
         };
 
     return {
+        _clear: function() {
```

Comment:
This deserves some explanation. TestTrackConfig is a singleton, and its private method `getConfig` memoizes the config loaded by ConfigParser. Because of the change to make the cookie name configurable, TestTrackConfig gets used a lot more frequently in the test suite, which exposed the fact that once getConfig is loaded and memoized, it can't be changed.

To properly test this class, I need to be able to change the config, so I'm creating this method just to clear out the config at the end of each test.

If anyone (@adamlangsner?) knows a better way to solve this problem, I'm all ears cuz I don't love this approach.